src/legacy/status_im/events.cljs

```diff
@@ -124,7 +123,7 @@
       :content             (i18n/label :t/biometric-auth-confirm-message)
       :confirm-button-text (i18n/label :t/biometric-auth-confirm-try-again)
       :cancel-button-text  (i18n/label :t/biometric-auth-confirm-logout)
-      :on-accept           #(biometric/authenticate nil {:on-fail on-biometric-auth-fail})
```

Comment:
Should this event not be moved out of "legacy"?

Reply:
Generally try to get things out of there, particularly if it's being used in the new ui. 
It's not limited to @flexsurfer , although he's been a big proponent in getting stuff moved from there/deleting old code. 
We should make a push soon to remove all references to legacy ns within status-im and try delete the legacy codebase completely.
Once it starts getting tangled it become a mess to maintain

Reply:
so here I guess I'd move out the whole event chain then

Reply:
If the effort is not too much that would be great! If it's a lot of work it can be better done in a follow up

Reply:
I think I'd leave it for a follow up 👍